source/MapPanel.cpp

```diff
@@ -1391,17 +1394,13 @@ void MapPanel::DrawSystems()
 		Point pos = zoom * (node.position + center);
 		RingShader::Draw(pos, OUTER, INNER, node.color);
 
-		if(commodity == SHOW_GOVERNMENT && node.government && node.government->GetName() != "Uninhabited")
+		if(commodity == SHOW_GOVERNMENT && node.government && node.government->GetName() != "Uninhabited" &&
+			pos.X() > left && pos.X() < right && pos.Y() > top && pos.Y() < bottom &&
+			(pow(pos.Y(), 2) + pow(pos.X(), 2)) < (pow((right + bottom) / 2, 2)))
```

Comment:
comment pls

Reply:
will do. I'm going to do some tweaking of the inclusion criteria first

Reply:
Done